frontend/database/i-wallet-database.ts

```diff
@@ -25,6 +25,22 @@ export abstract class IWalletDatabase {
     return JSON.parse(localStorage.getItem("network_type") || "null");
   }
 
+  getDbLocation(): "local" | "rxdb" | null {
+    return localStorage.dbLocation;
+  }
+
+  setDbLocation(value: "local" | "rxdb"): void {
```

Comment:
Lets add this difference to a enum of something like that. i case on the future we add more forms of storage.